### PR TITLE
changed lang

### PR DESCRIPTION
**Pull request by davislf2**
THIS IS A PULL REQUEST TEMPLATE! 
It helps you complete a pull request. Please follow the instructions below.

## Description

Tell us about your pull request!

## Changes made in this pull request
-
-
-

*Once your pull request is opened, please notify Josh.*

### CODE REVIEW

**Review by josh- — COMMENTED** [review]
😂 Thanks @davislf2 🐍

Can you please resolve the merge conflict before I merge?

File: participants/octocat.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 ![](https://github.com/octocat.png?s=150)
 
-I like programming in Ruby!
+I like programming in Python!
```
Comment (hugomd) [inline participants/octocat.md:5]:
```suggestion
I like programming in Elixir!
```

**Comment by ericjiang97** [comment]
As this PR change is getting a bit too heated as the debate between which progamming language is the best, I am closing this PR.

**Comment by ericjiang97** [comment]
I will be happy if you replace Ruby with something like `<language>`.

**Pull request closed without merging**